recipes-extended/fluentbit/files/td-agent-bit.conf

```diff
@@ -1,7 +1,7 @@
 [SERVICE]
     Flush     5
     Daemon    off
-    Log_Level info
+    Log_Level warning
```

Comment:
This is compile time thing, any way to set it runtime?

Reply:
I'd say this is a runtime thing? At least this configuration file is installed on the system